waspc/data/Generator/templates/react-app/src/queries/_query.js

```diff
@@ -2,14 +2,14 @@
 import { callOperation } from '../operations'
 import * as resources from '../operations/resources'
 
-const {= queryFnName =} = async (args) => {
-  return await callOperation('{= queryRoute =}', args)
+function {= queryFnName =}(args) {
+  return callOperation('{= queryRoute =}', args)
 }
 
 const queryCacheKey = '{= queryRoute =}'
 {= queryFnName =}.queryCacheKey = queryCacheKey
 
-export const entitiesUsed = {=& entitiesArray =}
+const entitiesUsed = {=& entitiesArray =}
```

Comment:
I wonder why I exported it. Maybe I thought it will be useful at some point to other parts of the system? But makes sense to make it private until we need it.